frontend/pages/Tools/[handle].tsx

```diff
@@ -50,6 +51,15 @@ export const getServerSideProps: GetServerSideProps<AppPageProps> = async (
       };
    }
 
+   if (productList.handle && productList.handle !== handle) {
+      return {
+         redirect: {
+            permanent: true,
+            destination: `/Tools/${encodeDeviceTitle(productList.handle)}`,
```

Comment:
I think the productList handle comes pre-encoded 
![image](https://user-images.githubusercontent.com/589425/182446204-609a847d-114a-4abe-a253-a4b5785d046b.png)

Reply:
Cool. Looking at `marketing` types, I'm gonna assume they come pre-encoded too.

Reply:
yes

Reply:
https://github.com/iFixit/react-commerce/pull/556/commits/f8c35cdd69403d62e810146fe6bf7ca5fa0a2141